src/noisepy/seis/plotting_modules.py

```diff
@@ -111,7 +118,7 @@ def plot_waveform(sfile, net, sta, freqmin, freqmax, savefig=False, sdir=None):
         if savefig:
             if not os.path.isdir(sdir):
                 os.mkdir(sdir)
-            outfname = sdir + "/{0:s}_{1:s}.{2:s}.pdf".format(sfile.split(".")[0], net, sta)
+            outfname = os.path.join(sdir, f"{raw_data_store}.{ts}_{net}.{sta}.pdf")
```

Comment:
One drawback of using the store is that we don't have a file name to use as a base name. But we also don't know what the `str` representation of `{raw_data_store}` will be. So I'm inclined to go with just this:
```suggestion
            outfname = os.path.join(sdir, f"{ts}_{net}.{sta}.pdf")
```
Since the ASDF files are named using the timespan, the behavior will be equivalent.

Reply:
done